src/main/scala/com/typesafe/sbt/packager/archetypes/JavaServerApplication.scala

```diff
@@ -234,14 +234,14 @@ object JavaServerAppPackaging extends AutoPlugin {
   }
 
   protected def startScriptMapping(name: String, script: Option[File], loader: ServerLoader, scriptDir: String): Seq[LinuxPackageMapping] = {
-    val (path, permissions) = loader match {
-      case Upstart => ("/etc/init/" + name + ".conf", "0644")
-      case SystemV => ("/etc/init.d/" + name, "0755")
-      case Systemd => ("/usr/lib/systemd/system/" + name + ".service", "0644")
+    val (path, permissions, isConf) = loader match {
+      case Upstart => ("/etc/init/" + name + ".conf", "0644", "true")
+      case SystemV => ("/etc/init.d/" + name, "0755", "false")
```

Comment:
Actually there were one or two minor things, certainly nothing worthy of delaying the release:

The `rpmVendor` setting is currently mandatory, but it seems like the `Vendor:` tag should now not be used.
See https://fedoraproject.org/wiki/How_to_create_an_RPM_package.

This could either be changed to a `Option[String]` setting, or just remove the code that does an `.isEmpty` check on the string?